ganttproject/src/biz/ganttproject/impex/google/GoogleAuth.java

```diff
@@ -0,0 +1,94 @@
+package biz.ganttproject.impex.google;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.extensions.java6.auth.oauth2.AuthorizationCodeInstalledApp;
+import com.google.api.client.extensions.jetty.auth.oauth2.LocalServerReceiver;
+import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeFlow;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.http.LowLevelHttpRequest;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.util.DateTime;
+
+import com.google.api.services.calendar.Calendar;
+import com.google.api.services.calendar.CalendarScopes;
+import com.google.api.services.calendar.model.*;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+public class GoogleAuth {
+
+  //Client ID
+  private static final String CLIENT_ID = "299608862260-vnb8d18h7cismqil9fbhf800jlnd3u6d.apps.googleusercontent.com";
+
+  //Client Secret
+  private static final String CLIENT_SECRET = "LUcqnHuf4NCqmEkljaJM0giT";
+
+  // Application name
+  private static final String APPLICATION_NAME =
+          "Google Export for GanttProject";
```

Comment:
these linebreaks here and below are unnecessary. Lines of length up to 100-120 symbols are ok